testomania/src/main/java/com/earth/testomania/home_screen/presentation/HomeScreen.kt

```diff
@@ -124,7 +119,7 @@ fun BottomContent(
     scope: CoroutineScope,
     navigator: DestinationsNavigator?
 ) {
-    when (pageType) {
+    /*when (pageType) {
```

Comment:
I thin we need to remove this or at least add TODO

Reply:
We need to fix it